Project3_STL/src/STLReader.cpp

```diff
@@ -0,0 +1,75 @@
+#include "../headers/STLReader.h"
+#include "../headers/Triangle.h"
+#include "../headers/Point3D.h"
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <vector>
+using namespace std;
+
+namespace Shapes3D

```

Comment:
Check if it works without using namespace block as it is already used in .h.

Reply:
I have updated cpp files as per your comment and it is working fine.